openc3/lib/openc3/utilities/authentication.rb

```diff
@@ -112,8 +114,8 @@ def _refresh_token(current_time)
       oath = _make_request(headers, data)
       @token = oath["access_token"]
       @refresh_token = oath["refresh_token"]
-      @expires_at = current_time + oath["expires_in"]
-      @refresh_expires_at = current_time + oath["refresh_expires_in"]
+      @expires_at = current_time + oath["expires_in"] - REFRESH_OFFSET_SECONDS
+      @refresh_expires_at = current_time + oath["refresh_expires_in"] - REFRESH_OFFSET_SECONDS
```

Comment:
Ahh we were previously letting the token go right down to the wire and then recreating? Do you think this resulted in a single bad un-authenticated request before the token was regenerated? Does this still have an issue where by if you don't make a request within 60s of the token expiring it will result in a single failure?